### PR TITLE
Updates SCMB examples replacing legacy code

### LINKED ISSUE #332: example scmb.py is broken with v4.x libray
### Scenario/Intent
tried running example/scmb.py with master version and it break at sec = security(con)

### Environment Details
 - **OneView SDK Version: master
 - **OneView Appliance Version:  3.1
 - **OneView Client API Version: 500
 - **Python Version: 3.5
 - **Platform: rhel 7.x

### Steps to Reproduce
try to run examples/scmb.py

### Expected Result
- examples should work with master version SDK library v4.x 
- Even though i'm raising issue with scmb.py but all examples should be updated to make it with current version of library
 - It might be good idea to provide support for backward compatibility or document it

### Actual Result
getting python error:
NameError: name 'security' is not defined

from scmb.py line # 282 ( which is sec = security(con)

### PR DESCRIPTION
**Pull request by tmiotto**
### Description
Replaces non-functional legacy code in scmb examples

### Issues Resolved
Fixes #332 

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Comment by tmiotto** [comment]
@fgbulsoni  Yes and No.

Yes we already do have a resource for RabbitMQ certificates.

No, we don't want to add any other action to RabbitMQ resource. First because the methods are simply handling certificates and there are other methods creating connections using amqp. Also, the scmb examples README makes it clear that it is an example on how to connect the SDK to an external tool.

File: examples/scmb/scmb.py [hunk 1]
```diff
@@ -113,39 +113,34 @@ def recv(host, route):
...
-def login(con, credential):
-    # Login with givin credentials
-    try:
-        con.login(credential)
-    except:
-        print('Login failed')
-
-
-def acceptEULA(con):
+def acceptEULA(oneview_client):
```
Comment (fgbulsoni) [inline examples/scmb/scmb.py]: Should add an extra blank line before this method

**Comment by fgbulsoni** [comment]
Weirdly, I'm also getting these other issues on the ov_to_sm example:
```
Warning	Flake8	F821 — undefined name 'http'	43:20
Warning	Flake8	F821 — undefined name 'http'	54:16
Warning	Flake8	F821 — undefined name 'time'	57:13
Warning	Flake8	F821 — undefined name 'common'	115:20
```

Is it perhaps my environment? I see travis's flake8 did not hit those. But at the same time I don't see where those are being declared.

**Comment by fgbulsoni** [comment]
AH, travis is ignoring `examples/scmb/`, I'd suggest fixing those up and checking line 14 on `tox.ini`:
`exclude = tests.py, hpOneView/__init__.py, examples/scmb/, examples/scripts`

I believe `examples/scripts` has already been dropped, `examples/scmb/` should be passing with this PR and I haven't seen `tests.py`, so those three could likely be removed from the `exclude`

File: tox.ini [hunk 2]
```diff
@@ -11,7 +11,7 @@ skip_missing_interpreters = true
 [flake8]
 ignore = E402
 max-line-length = 140
-exclude = tests.py, hpOneView/__init__.py, examples/scmb/, examples/scripts
+exclude = tests.py, hpOneView/__init__.py, examples/scripts
```
Comment (fgbulsoni) [inline tox.ini]: examples/scripts should be dropped as well, the folder does not exist

File: examples/scmb/scmb.py [hunk 3]
```diff
@@ -22,7 +22,7 @@
 # THE SOFTWARE.
 ###
 
-from hpOneView import *
+from hpOneView.oneview_client import OneViewClient
 from functools import partial
```
Comment (fgbulsoni) [inline examples/scmb/scmb.py]: Suggestion: Perhaps add a blank link to separate `from X import X` and `import X` types of imports

**Comment by tmiotto** [comment]
Fixes #332

**Pull request merged**